apps/codeforafrica/src/payload/collections/Users.js

```diff
@@ -0,0 +1,58 @@
+import { isAdmin, isAdminFieldLevel } from "../access/isAdmin";
+import {
+  isAdminOrSelf,
+  isAdminOrSelfFieldLevel,
+} from "../access/isAdminOrSelf";
+import { ROLE_DEFAULT, ROLE_OPTIONS } from "../access/roles";
+
+const Users = {
+  slug: "users",
+  access: {
+    create: isAdmin,
+    read: () => true,
+    update: isAdminOrSelf,
+    delete: isAdminOrSelf,
+  },
+  admin: {
+    defaultColumns: ["firstName", "lastName", "email", "updatedAt"],
+    enableRichTextLink: false,
+    group: "Website",
+    useAsTitle: "email",
+  },
+  auth: {
+    verify: true,
+  },
+  fields: [
+    {
+      type: "row",
+      fields: [
+        {
+          name: "firstName",
+          type: "text",
+          required: true,
+        },
+        {
+          name: "lastName",
+          type: "text",
+          required: true,
+        },
+      ],
+    },
+    {
+      name: "roles",
+      type: "select",
+      // Save this field to JWT so we can use from `req.user`
+      saveToJWT: true,
+      required: true,
+      defaultValue: [ROLE_DEFAULT],
```

Comment:
@kelvinkipruto This shouldn't be an array since hasMany is false. Pushing a fix.

Reply:
I think we should turn `hasMany` back to `true` @koechkevin / @kelvinkipruto . If you look at [access methods](https://github.com/CodeForAfrica/ui/blob/6783676785229949c041355282dde1f3e25ed41e/apps/codeforafrica/src/payload/access/isAdminOrSelf.js#L7), they expect roles to be an array.

... not sure what I was testing there.